### PR TITLE
Neo xrefs

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

File: modules/project-docs/partials/attributes.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:ruby-api-link: https://docs.couchbase.com/sdk-api/couchbase-ruby-client-3.2.0/index.html
-:ruby-current-version: 3.2.0
+:ruby-api-link: https://docs.couchbase.com/sdk-api/couchbase-ruby-client/index.html
+:ruby-current-version: 4.0.0
```
Comment (osfameron) [inline modules/project-docs/partials/attributes.adoc]: Shouldn't this be 3.3.0?

**Review by osfameron — COMMENTED** [review]
Looks ok apart from 4.0/3.3 question?

File: modules/project-docs/partials/attributes.adoc (hunk 1 above)
Comment (maria-robobug) [reply modules/project-docs/partials/attributes.adoc]: yep, that's a mistake 👍

Comment (RichardSmedley) [reply modules/project-docs/partials/attributes.adoc]: Yes. Yes it should.

**Pull request merged**